apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -58,9 +60,9 @@ export const askAITutor = createAsyncThunk(
     };
 
     const tutorType = chatContext.tutorType;
-    const generalChat = tutorType === TutorType.GENERAL_CHAT;
-    const compilation = tutorType === TutorType.COMPILATION;
-    const validation = tutorType === TutorType.VALIDATION;
+    const generalChat = tutorType === TutorTypes.GENERAL_CHAT;
```

Comment:
I got a little refactor happy and renamed TutorType to TutorTypes to indicate that this references an object that stores all of the types/multiple fields 😅